### ISSUE TITLE
Sign in does not work

### ISSUE THREAD

**Issue by Ceebox** [LibreVR/Revive#1521]
Revive keeps prompting me to log in, but when I log in it says invalid, and when the prompt box comes back it has changed the username I put in and also the password.

Open Revive and try to log in, it is supposed to log you in, but it does not work.

**Expected behavior**

I'm running WMR and the latest build of windows 10.

 - Revive: Latest release
 - SteamVR: Latest

**Comment by ahmanwoods** [comment]
Do you have 2FA enabled?

**Comment by Ceebox** [comment]
@ahmanwoods No, no 2FA. No clue why it is being like this. It always returns the name of my PC and then a / for my name.

**Comment by EthanZoneCoding** [comment]
Same issue here. I am certain my info was correct

**Comment by JTwinleaf** [comment]
Same issue except every time I attempt to log in Revive seems to crash
